question/type/random/questiontype.php

```diff
@@ -130,59 +131,12 @@ public function get_question_options($question) {
     /**
      * Random questions always get a question name that is Random (cateogryname).
      * This function is a centralised place to calculate that, given the category.
-     * @param stdClass $category the category this question picks from. (->parent, ->name & ->contextid are used.)
-     * @param bool $includesubcategories whether this question also picks from subcategories.
-     * @param string[] $tagnames Name of tags this question picks from.
+     * @param stdClass $slotdata the category this question picks from. (->parent, ->name & ->contextid are used.)
      * @return string the name this question should have.
      */
-    public function question_name($category, $includesubcategories, $tagnames = []) {
```

Comment:
why this is changed? random qtype is not used anymore, if this method is used, this is wrong, this plugin is getting removed in 4.1